### ISSUE TITLE
Please create GitHub tags for new releases

### ISSUE THREAD

**Issue by eLvErDe** [nesdis/djongo#498]
Thanks !

**Comment by nesdis** [comment]
Yes. It is a work in progress. Will start this practice henceforth.

**Comment by eLvErDe** [comment]
Great.

I'm planning to get djongo into an official Debian package but sadly the PyPi tarball misses docs and tests which is definitely required !
